.travis.yml

```diff
@@ -1,9 +1,25 @@
+sudo: true
+dist: trusty
 language: node_js
 node_js:
-  - "6"
-  - "7"
+- '6'
+- '7'
 before_script:
-  - npm install
+- npm install
 script:
-  - npm test
+- npm test
 after_success: npm run coverage
+jobs:
+  include:
+  - stage: build docker image
+    script:
+    - docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+    - docker build --build-arg=http_proxy=$http_proxy --build-arg=https_proxy=$https_proxy -t docker.io/jesseolsen/hpe-oneview-hubot:latest
```

Comment:
oops...  remove the http_proxy and https_proxy... those are only for my local test builds...
Also, not sure if the '.' on line 18 is intentional....